tests/Util/yaml_fixtures/array_double_quoted_value.test

```diff
@@ -0,0 +1,14 @@
+caddy:
+    ports:
+      - "80:80"
```

Comment:
For purposes of figuring out the root cause, if you changed this from `"80:80"` to `"foo_bar"`, does the error happen?

Or, if you use `'80:80'` with single quotes - error or no?